src/game/scenes/Game.ts

```diff
@@ -38,18 +39,32 @@ export class Game extends Phaser.Scene {
 		if (tileset) {
 			// Create layers from the tilemap
 			this.colliderLayers = map.createLayer("layer", tileset, 0, 0);
-			this.player = this.physics.add.sprite(
+
+			// Create the player sprite
+			const playerSprite = this.physics.add.sprite(
 				GAME_OPTIONS.gameSize.width / 2,
 				GAME_OPTIONS.gameSize.height / 2,
 				SpritesPlayer.getName(),
 			);
```

Comment:
imo, we should handle the sprite creation inside the `PlayerCharacter` class and just pass the sprite key (string)